src/Utils.js

```diff
@@ -1,3 +1,78 @@
 (function(){
   LifeGamification.utils = {};
+  LifeGamification.work = {};
+  LifeGamification.refreshTimer;
+
+  LifeGamification.utils.setStartTime = function () {
+    return new Promise((resolve, reject) => {
+      LifeGamification.repository.getWork()
+      .then(function(result){
+        if(result){
+          LifeGamification.work = result;
+          resolve();
+        }
+      });
+    });
+  }
+
+  LifeGamification.utils.startTiming = function () {
+    if(!($('.timer__select-skill').val())){
+      console.log("Error: Add a skill first.");
+      return;
+    }
+    let currentTime = new Date().getTime();
+    let skillName = $('.timer__select-skill').val();
+    LifeGamification.repository
+      .setWork({startTime: currentTime, name: skillName})
+    .then(LifeGamification.utils.handleTimer());
+  }
+
+  LifeGamification.utils.endTiming = function () {
+    const skill = LifeGamification.
+      skillsCollection[LifeGamification.work.name];
+    const expToAdd = Math.floor(calcTime() / 60);
+    LifeGamification.models.updateExp(skill, expToAdd);
+
+    LifeGamification.repository.setWork({});
+    clearInterval(LifeGamification.refreshTimer);
+    LifeGamification.view.finishTimer();
+  }
+
+  displayTime = function(time) {
+    let hours = Math.floor(time / 3600);
```

Comment:
If you sure that there aren't any other assigments for `hours` variable, you should use `const` instead of `let`.

Reply:
There are: I add prefix "0". Should I add another variable instead?